src/core/openthread.cpp

```diff
@@ -991,6 +994,25 @@ void HandleActiveScanResult(void *aContext, Mac::Frame *aFrame)
     return;
 }
 
+ThreadError otEnergyScan(uint32_t aScanChannels, uint16_t aScanDuration, otHandleEnergyScanResult aCallback,
```

Comment:
@luwannest, many of these query/request commands are specific to the commissioner role and should not be a part of the core.  Instead, I think they can be implemented as part of the commissioner role of #399.  I'm inclined to merge #399 sooner rather than later so that we can parallelize work on the commissioner role.

Have you started working on the PAN ID Conflict notification (#434)?  If not, I can implement that to give a better idea of my thoughts.